tyssue/solvers/quasistatic.py

```diff
@@ -133,3 +132,105 @@ def check_grad(self, eptm, geom, model):
             self._opt_energy, self._opt_grad, pos0.flatten(), eptm, geom, model
         )
         return grad_err
+
+    # The functions bellow are for a perodic square tissue in 2D
+    def find_energy_min_periodic_square_tissue(self, eptm, geom, model, **minimize_kw):
```

Comment:
Maybe this can instead be an option in the main find_energy_min() function, defaulting to `False`:
```py
find_energy_min(self, eptm, geom, model, periodic=False, **minimize_kw)
```

And switch between ''regular'' and periodic with an if... It's easier for the user to adapt existing code.

Reply:
Yes that sounds good and I implemented it.